packages/template-retail-react-app/package.json

```diff
@@ -77,7 +80,8 @@
     "react-helmet": "^6.1.0",
     "react-hook-form": "^7.43.9",
     "react-intl": "^5.25.1",
-    "react-router-dom": "^5.3.4"
+    "react-router-dom": "^5.3.4",
+    "shelljs": "^0.8.5"
```

Comment:
if we have those scripts at root, one less dependency for the generated project ; )